osbs/build/build_request.py

```diff
@@ -234,7 +240,9 @@ def set_secret_for_plugin(self, plugin, secret):
                         'mountPath': secret_path,
                     })
 
-                self.dj.dock_json_set_arg(*(plugin + (secret_path,)))
+                # tag_and_push sets secret path for each registry separately
```

Comment:
Just spotted this comment. Should probably be 'if no argument name supplied, leave config alone'.